From 23aeaa62217acd281ac4cfe23e16ab2805139c22 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 24 Feb 2022 10:06:13 +0200
Subject: [PATCH] [SW-0] thermal: remove unused header

The removed header is not used, and it also potentially breaks
common code because it's a device specific header.

Change-Id: I2fbd7a63f428bb0cc00f0b162f375b68e72f43ea
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/lib/preboot/thermal_v2.c b/lib/preboot/thermal_v2.c
index 681ff3d..380bb7f 100644
--- a/lib/preboot/thermal_v2.c
+++ b/lib/preboot/thermal_v2.c
@@ -5,8 +5,6 @@
  *
  */
 
-#include <psoc_global_conf_regs.h>
-
 #include <preboot.h>
 #include <pll.h>
 #include <uart.h>
